pages/docs/couple-your-code/couple-your-code-porting-v2-3.md

```diff
@@ -118,6 +118,7 @@ Please add breaking changes here when merged to the `develop` branch.
   - Directly define face elements or cells of your coupling mesh available in your solver by passing their vectices to preCICE, which automatically handles edges of triangles etc. See [Mesh Connectivity](couple-your-code-defining-mesh-connectivity) for more information.
   - Rename `setMeshTriangleWithEdges` to `setMeshTriangle` and `setMeshQuadWithEdges` to `setMeshQuad`. The edge-based implementation was removed.
   - Use the new bulk functions to reduce sanitization overhead: `setMeshEdges`, `setMeshTriangles`, `setMeshQuads`, `setMeshTetrahedra`.
+- Replace pointer data structures by vectors, e.g., `double* forces = new double[vertexSize*dim]` to `std::vector<double> forces(vertexSize*dim)`.
```

Comment:
In this example, `forces` is a data structure create and handled by the user. The way we used it in our example API was just one version of how such a data structure could look like and how to use it with preCICE. However, both versions (i.e. what you call pointer data structure and vectors) work with the v2 API as well as the v3 API. The C-style array we use in our v2 example is not really a C++ way of doing things and I would expect C++ codes to use vectors anyway. Having this in mind, I am not sure about this hint in the porting guide. We should mention somewhere that the signature (of e.g. readData) changed from a raw pointer to a `span`.

Reply:
Okay, I agree that using raw pointers is just one specific case.
> We should mention somewhere that the signature (of e.g. readData) changed from a raw pointer to a span.

Stating this should be sufficient, since this already clarifies that raw pointers can no longer be an argument to `readData`.


In any case my motivation for this was a case in v2 of a `std::vector<double> forces` that was given to `readData` as `forces.data()`. So when I ported to v3 to me it was not obvious that I should drop the `data.()`.